dart/lib/src/enricher/io_enricher_event_processor.dart

```diff
@@ -100,9 +105,8 @@ class IoEnricherEventProcessor extends EventProcessor {
 
   SentryDevice _getDevice(SentryDevice? device) {
     return (device ?? SentryDevice()).copyWith(
-      language: device?.language ?? Platform.localeName,
       name: device?.name ?? Platform.localHostname,
-      timezone: device?.timezone ?? DateTime.now().timeZoneName,
```

Comment:
Removing the setters would be a breaking change and it should make to v7 then.
It'd be fine to keep setting them, mark them as deprecated, but also setting in the culture object (so we send in both fields), and later remove the deprecated fields.
What do you think about that?